### PR TITLE
Feature/kitosudv 1942 references

### PR DESCRIPTION
**Pull request by mrjsawdk**
## KITOS Pull request template
The following procedure dictates the steps needed before a Pull request can be merged into master.

- [x] **Implement**: 
      *All requirements are implemented and unit tests are green*
      
- [x] **Merge master into branch / rebase with master**: 
      *Make sure you are testing your changes and how they co-exist with the latest version of master*
      
- [x] **Green on integration**: 
      *All integration tests are green on integration*
      
- [x] **Request review**: 
      *Tag whomever you wish to review your code*
      
- [x] **Review completed**: 
      *Reviewer ticks this box when review comments have been submitted*
      
- [x] **Changes**: 
      *PR owner and reviewer agrees on which changes must be made and the changes are committed.*
      
- [x] **Merge master into branch / rebase with master**: 
      *Make sure you are testing your changes and how they co-exist with the latest version of master*
      
- [x] **Green on integration**: 
      *All integration tests are green on integration*

### CODE REVIEW

File: Tests.Unit.Core.ApplicationServices/ApplicationServices/SystemUsage/ItSystemUsageWriteServiceTest.cs [hunk 1]
```diff
@@ -794,7 +799,136 @@ public void Cannot_Create_With_KLEDeviations_If_RemovedKLE_Do_Not_Exist_On_Syste
...
+        {
+            //Arrange
+            var (systemUuid, organizationUuid, transactionMock, organization, itSystem, itSystemUsage) = CreateBasicTestVariables();
+
+            SetupBasicCreateThenUpdatePrerequisites(organizationUuid, organization, systemUuid, itSystem, itSystemUsage);
+            Configure(f => f.Inject(false));
+            var externalReferences = Many<UpdatedExternalReferenceProperties>().ToList();
+            
+            //Set two masters
+            foreach (var master in externalReferences.Take(2))
```
Comment (JacobMalling) [inline Tests.Unit.Core.ApplicationServices/ApplicationServices/SystemUsage/ItSystemUsageWriteServiceTest.cs:868]: I stedet for 2 hvad så med en tilfældig værdi mellem 2 og externalReferences.Count()?

File: Tests.Unit.Core.ApplicationServices/ApplicationServices/SystemUsage/ItSystemUsageWriteServiceTest.cs [hunk 2]
```diff
@@ -794,7 +799,136 @@ public void Cannot_Create_With_KLEDeviations_If_RemovedKLE_Do_Not_Exist_On_Syste
...
+        }
+
+        [Fact]
+        public void Cannot_Create_With_ExternalReferences_And_Multiple_Masters()
+        {
+            //Arrange
+            var (systemUuid, organizationUuid, transactionMock, organization, itSystem, itSystemUsage) = CreateBasicTestVariables();
+
+            SetupBasicCreateThenUpdatePrerequisites(organizationUuid, organization, systemUuid, itSystem, itSystemUsage);
+            Configure(f => f.Inject(false));
```
Comment (JacobMalling) [inline Tests.Unit.Core.ApplicationServices/ApplicationServices/SystemUsage/ItSystemUsageWriteServiceTest.cs:864]: Kan det passe at denne bruges for at "MasterReference: bool" bliver false i alle de oprettede?

**Review by JacobMalling — APPROVED** [review]
Lige en enkelt mindre ting

File: Tests.Unit.Core.ApplicationServices/ApplicationServices/SystemUsage/ItSystemUsageWriteServiceTest.cs (hunk 1 above)
Comment (mrjsawdk) [reply Tests.Unit.Core.ApplicationServices/ApplicationServices/SystemUsage/ItSystemUsageWriteServiceTest.cs:873]: tjo boundary er jo > 1 så det er egentlig derfor. Vi tester lige på og på den ene og anden side af kanten. Alt derudover er ligemeget

File: Tests.Unit.Core.ApplicationServices/ApplicationServices/SystemUsage/ItSystemUsageWriteServiceTest.cs (hunk 2 above)
Comment (mrjsawdk) [reply Tests.Unit.Core.ApplicationServices/ApplicationServices/SystemUsage/ItSystemUsageWriteServiceTest.cs:869]: ja det er fordi jeg bruger autofixture bagefter til at lave Many--- jeg tilføjer en comment

**Pull request merged**